pkg/lib/indexer/indexer.go

```diff
@@ -176,6 +179,8 @@ type PruneStrandedFromIndexRequest struct {
 	FromIndex         string
 	OutDockerfile     string
 	Tag               string
+	CaFile            string
+	SkipTLS           bool
```

Comment:
This has got me thinking: do we want to set a custom CA and TLS settings per-request, or for all requests an indexer makes? I don't think we need to change this here, but worth remember for future refactoring.